src/ripple/protocol/STParsedJSON.h

```diff
@@ -45,7 +45,7 @@ class STParsedJSONObject
     ~STParsedJSONObject () = default;
 
     /** The STObject if the parse was successful. */
-    std::unique_ptr <STObject> object;
+    boost::optional <STObject> object;
```

Comment:
@ximinez Trailing underscores are only for private data members. Also, for cleanups like this we should do them ourselves and let David spend his time doing more valuable things.